etl/run_python_step.py

```diff
@@ -21,7 +21,7 @@ def main(uri: str, dest_dir: str, ipdb: Optional[bool]) -> None:
     Import and run a specific step of the ETL. Meant to be ran as
     a subprocess by the main `etl` command.
     """
-    if not uri.startswith("data://") and not uri.startswith("data-private://"):
+    if not uri.startswith("data://") and not uri.startswith("data-private://") and not uri.startswith("upsert://"):
```

Comment:
I can't see any reference to `upsert://` steps using this path. Were you experimenting?

Reply:
These are dynamically generated from datasets in the grapher channel.